From 8dbeec103124f9de1ed515ef50ff79eff3ecd52c Mon Sep 17 00:00:00 2001
From: Sergey V. Udaltsov <svu@gnome.org>
Date: Sun, 25 Apr 2010 22:59:53 +0000
Subject: Provide the proper place in NA by using the name

The name "keyboard" is known to the NA applet - so it positions the icon
properly, on the right side, after all apps
---
diff --git a/libgnomekbd/gkbd-status.c b/libgnomekbd/gkbd-status.c
index efcad6a..5e62ae9 100644
--- a/libgnomekbd/gkbd-status.c
+++ b/libgnomekbd/gkbd-status.c
@@ -632,6 +632,9 @@ gkbd_status_init (GkbdStatus * gki)
 
 	gki->priv = g_new0 (GkbdStatusPrivate, 1);
 
+	/* This should give NA a hint about the order */
+	gtk_status_icon_set_name (GTK_STATUS_ICON(gki), "keyboard");
+
 	xkl_debug (100, "Initiating the widget startup process for %p\n",
 		   gki);
 
--
cgit v0.8.3.1
